app/app.go

```diff
@@ -886,6 +895,9 @@ func (app *App) BuildDependencyDag(ctx sdk.Context, txs [][]byte) (*Dag, error)
 		}
 		msgs := tx.GetMsgs()
 		for messageIndex, msg := range msgs {
+			if isGovMessage(msg) {
+				return nil, ErrGovMsgInBlock
+			}
```

Comment:
Hmm, it's not completely an error since this is expected right?

Reply:
yeah true, but this seemed like the best way to interrupt the dependency graph creation and have the sequential behavior. any idea on a better pattern to use?

Reply:
Yeah, that's fair -- perhaps we could just leave a comment here noting why gov messages are handled this way